code/modules/mob/living/carbon/carbon.dm

```diff
@@ -501,8 +501,6 @@
 	update_stat()
 	if(((maxHealth - total_burn) < HEALTH_THRESHOLD_DEAD) && stat == DEAD )
 		become_husk()
-		if(on_fire)
-			shred_clothing()

```

Comment:
Your clothes don't automatically shred when you become a husk (was especially weird for spaced bodies).